keyboards/preonic/keymaps/kyrias/keymap.c

```diff
@@ -0,0 +1,192 @@
+/* Copyright 2015-2021 Jack Humbert
+ * Copyright 2022-2023 Johannes Löthberg (@kyrias)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#include "config.h"
+
+enum preonic_keycodes {
+  DVORAK = SAFE_RANGE,
```

Comment:
```suggestion
  DVORAK = QK_USER,
```

Reply:
The PR checklist says to use `= SAFE_RANGE`, has this requirement changed without the documentation being updated?

Reply:
`SAFE_RANGE` is a bit of a mess.  right now, it aliases to `QK_KB`, which is means for keyboards, not keymaps.  `QK_USER` is what should be used for keymaps, and is more explicit.